assets/evaluation_results/squad_v2_databricks-dbrx-base_question_answering/spec.yaml

```diff
@@ -0,0 +1,33 @@
+type: evaluationresult
+name: squad_v2_databricks-dbrx-base_question_answering
+version: 2.23.05
+display_name: databricks-dbrx-base_squad_v2_text_generation
+description: databricks-dbrx-base run for squad_v2 dataset
+dataset_family: squad_v2
+dataset_name: squad_v2
+
+model_name: databricks-dbrx-base
+model_version: "3"
+model_asset_id: azureml://registries/azureml-staging/models/databricks-dbrx-base/versions/3
```

Comment:
Is there any specific reason we have chosen the model from `azureml-staging` registry? Ideally it should be from one of the prod registries. Let's verify the `model_name`, `model_version` and `model_asset_id` for all the models.